specification/contosowidgetmanager/Contoso.WidgetManager/main.tsp

```diff
@@ -1,21 +1,22 @@
-import "@cadl-lang/rest";
-import "@cadl-lang/versioning";
-import "@azure-tools/cadl-azure-core";
+import "@typespec/rest";
```

Comment:
Http has been split out from rest. So need to explicitly `import "@typespec/http"`.

Reply:
I added it and it compiles successfully. It doesn't seem to be needed though.